packages/core/src/electron-browser/menu/electron-menu-contribution.ts

```diff
@@ -65,12 +66,25 @@ export namespace ElectronMenus {
     export const FILE_CLOSE = [...CommonMenus.FILE_CLOSE, 'window-close'];
 }
 
+export namespace ZoomLevelConstraints {
```

Comment:
I do not believe anyone is using `ZoomLevelConstraints` outside the file or class.
You can these constants directly in the `ElectronMenuContribution` class, with documentation to the link of where the values are derived from (the preferences).